testsuite/features/profiles/Docker/Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM registry.mgr.suse.de/toaster-sles12sp3-products
-MAINTAINER Dario Maiocchi "dmaiocchi@suse.com"
+FROM registry.mgr.suse.de/toaster-sles12sp4-products
```

Comment:
@dincamihai does a toaster container with this name exists on registry.mgr.suse.de?

Reply:
Waiting for @dincamihai 's answer.

Reply:
@Bischoff we don't have the sp4 yet

Reply:
OK, then i'll revert that change until you have it @dincamihai .